conformance/tests/httproute-invalid-cross-namespace-parent-ref.go

```diff
@@ -37,8 +39,14 @@ var HTTPRouteInvalidCrossNamespaceParentRef = suite.ConformanceTest{
 		gwNN := types.NamespacedName{Name: "same-namespace", Namespace: "gateway-conformance-infra"}
 		routeNN := types.NamespacedName{Name: "invalid-cross-namespace-parent-ref", Namespace: "gateway-conformance-web-backend"}
 
-		t.Run("Route should not have Parents set in status", func(t *testing.T) {
-			kubernetes.HTTPRouteMustHaveNoAcceptedParents(t, suite.Client, suite.TimeoutConfig, routeNN)
+		t.Run("HTTPRoute should have an Accepted: false condition with reason NotAllowedByListeners", func(t *testing.T) {
```

Comment:
So we've removed the check for `kubernetes.HTTPRouteMustHaveNoAcceptedParents(t, suite.Client, suite.TimeoutConfig, routeNN)` but isn't that check still valid, even if we also add the check to ensure the status was updated correct? :thinking:

Reply:
Fair enough, I thought it was mostly redundant since it was checking for a parent with an "Accepted: false" condition (with any reason), but I guess it does also validate that no _other_ parents accept the route. Will add back

Reply:
Sounds good, thank you :+1: